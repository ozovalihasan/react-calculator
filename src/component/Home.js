import React from 'react';

const Home = () => (
  <div className="home main">
    <div className="home title">Welcome to our page</div>
    <div className="home subtitle">
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Tempora
      repellendus iste, reprehenderit, quam dignissimos explicabo aperiam
      quisquam voluptatibus molestias aliquam natus commodi necessitatibus cum
      quibusdam sapiente ipsa assumenda aut eius!
    </div>
    <div className="home subtitle">
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Tempora
      repellendus iste, reprehenderit, quam dignissimos explicabo aperiam
      quisquam voluptatibus molestias aliquam natus commodi necessitatibus cum
      quibusdam sapiente ipsa assumenda aut eius!
    </div>
  </div>
);
export default Home;
