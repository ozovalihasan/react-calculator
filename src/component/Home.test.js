import React from 'react';
import { render, unmountComponentAtNode } from 'react-dom';
import { act } from 'react-dom/test-utils';
import Home from './Home';

let container = null;

beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

it('shows static text ', () => {
  act(() => {
    render(<Home />, container);
  });
  expect(container.textContent).toBe(
    'Welcome to our pageLorem, ipsum dolor sit amet consectetur adipisicing elit. Tempora repellendus iste, reprehenderit, quam dignissimos explicabo aperiam quisquam voluptatibus molestias aliquam natus commodi necessitatibus cum quibusdam sapiente ipsa assumenda aut eius!Lorem, ipsum dolor sit amet consectetur adipisicing elit. Tempora repellendus iste, reprehenderit, quam dignissimos explicabo aperiam quisquam voluptatibus molestias aliquam natus commodi necessitatibus cum quibusdam sapiente ipsa assumenda aut eius!',
  );
});
