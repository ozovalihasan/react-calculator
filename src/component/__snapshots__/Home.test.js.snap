// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="home main"
>
  <div
    className="home title"
  >
    Welcome to our page
  </div>
  <div
    className="home subtitle"
  >
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Tempora repellendus iste, reprehenderit, quam dignissimos explicabo aperiam quisquam voluptatibus molestias aliquam natus commodi necessitatibus cum quibusdam sapiente ipsa assumenda aut eius!
  </div>
  <div
    className="home subtitle"
  >
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Tempora repellendus iste, reprehenderit, quam dignissimos explicabo aperiam quisquam voluptatibus molestias aliquam natus commodi necessitatibus cum quibusdam sapiente ipsa assumenda aut eius!
  </div>
</div>
`;
